src/lib/api/coaching-sessions.ts

```diff
@@ -25,22 +25,35 @@ export const CoachingSessionApi = {
    * coaching sessions should be retrieved from.
    * @param fromDate A date specifying the earliest coaching session date to return.
    * @param toDate A date specifying the latest coaching session date to match.
+   * @param sortBy Optional field to sort by (date, created_at, updated_at).
+   * @param sortOrder Optional sort order (asc, desc).
    * @returns Promise resolving to an array of CoachingSession objects (empty array if data is not yet loaded or
    *  relationshipId is null)
    */
   list: async (
     relationshipId: Id | null,
     fromDate: DateTime,
-    toDate: DateTime
+    toDate: DateTime,
+    sortBy?: 'date' | 'created_at' | 'updated_at',
+    sortOrder?: 'asc' | 'desc'
```

Comment:
Similar to the BE PR, would it make sense to centralize a type like this since we'll 100% be using it elsewhere?

since we're using the `sortBy?: 'date' | 'created_at' | 'updated_at',` below in this file as well we could maybe extract that into a type too? Even if that one is just local to this file